README-dev.md

```diff
@@ -1,22 +1,31 @@
 # o1js README-dev
 
-o1js is a TypeScript framework designed for zk-SNARKs and zkApps on the Mina blockchain.
+o1js is a TypeScript framework designed for zk-SNARKs and zkApps on the Mina
```

Comment:
Are these line-length diffs intentional, or just your local IDE settings?

Reply:
These are intentional, yes.
I added these changes in two independent commits that can be removed if there is a strong opinion on not enforcing a limit of 80 characters.

It is quite commonly accepted to keep 80 characters as a limit, not only for historical reasons, but also because it is easier to go through files in modern editors too when we have multiple buffers open on the same screen.

Reply:
(but also my editor is configured to make it relatively automatic when I open a file)